docs/extensions.md

```diff
@@ -311,7 +311,16 @@ class SomeBlocks {
 
                 // Dynamic menu: returns an array as above.
                 // Called each time the menu is opened.
-                menuB: 'getItemsForMenuB'
+                menuB: 'getItemsForMenuB',
+
+                // The examples above are shorthand for setting only the `items` property in this full form:
+                menuC: {
+                    // This flag makes a "non-droppable" menu: the menu will not accept reporters.
+                    rejectReporters: true,
```

Comment:
Frankly, save for someone hypothetically confusing the term with "reject" in Promise lingo, I think `rejectReporters` isn't too bad.. :P

Another option would be something along the lines of "only" - `onlyDropdown`, `onlyField`, etc. But I think `rejectReporters` is more clear than those. There's also `disallowReporters`, but that sort of screams, "It *should* be `allowReporters: false`!" - negation in the value rather than the name, that is.

PS a fun thing with making menus be in object form is that we can add entries like `changeCallback`, for implementing things like the "stop (all/this script/other scripts in sprite)" block!